boa/src/syntax/ast/op.rs

```diff
@@ -680,8 +680,8 @@ pub enum LogOp {
     ///  - [ECMAScript reference](
     ///  - [MDN documentation][mdn]
     ///
-    /// [spec]: https://tc39.es/ecma262/#prod-LogicalORExpression)
-    /// [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_Operators#Logical_OR
+    /// [spec]: <https://tc39.es/ecma262/#prod-LogicalORExpression)>
+    /// [mdn]: <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_Operators#Logical_OR>
```

Comment:
I believe this is not correct, right? It should show `[spec]` next to `[ECMAScript reference]`, and then, the links should not have `<...>`. Also, the `spec` link should not have a closing `)`